tsl/test/sql/compression_errors.sql

```diff
@@ -97,6 +97,9 @@ ALTER TABLE foo RESET (timescaledb.compress);
 ALTER TABLE foo ADD CONSTRAINT chk CHECK(b > 0);
 ALTER TABLE foo ADD CONSTRAINT chk UNIQUE(b);
 ALTER TABLE foo DROP CONSTRAINT chk_existing;
+--can add index , but not unique index
+CREATE UNIQUE INDEX foo_idx ON foo ( a, c );
+CREATE INDEX foo_idx ON foo ( a, c );
```

Comment:
You have three alternatives when it should not be possible to create an index, but here you test only one. Suggest to add tests for the other two cases.

Reply:
Could you elaborate on what you mean by 3 alternatives?

Reply:
Yes, the if condition looks like this:

```
stmt->unique || stmt->primary || stmt->isconstraint
```

AFAICT, these correspond to a UNIQUE index, a check constraint, and a primary key, but you only create a unique index.

Reply:
As we discussed, there were checks for the other cases in other parts of the code.